napari/_app_model/context/_layerlist_context.py

```diff
@@ -130,6 +130,10 @@ def _active_is_image_3d(s: LayerSel) -> bool:
     )
 
 
+def _empty_shapes_layer_selected(s: LayerSel) -> bool:
+    return any(x._type_string == "shapes" and not x.data.size for x in s)
```

Comment:
```suggestion
    return any(x._type_string == "shapes" and not len(x.data) for x in s)
```

My bad. I do not notice that for this layer, the type for data is list.

Reply:
Ups, I should have also read more thoroughly the review suggestions before applying them 😅 

Thank you for the review @Czaki !